src/bun.js/api/BunObject.zig

```diff
@@ -2453,12 +2536,31 @@ pub const Crypto = struct {
                 return null;
             }
 
+            const this = bun.default_allocator.create(CryptoHasher) catch return null;
```

Comment:
We do not need to allocate until the data is validated successfully. Ideally, error conditions wouldn't allocate